README.md

```diff
@@ -1 +1,3 @@
-# jslings
\ No newline at end of file
+# jslings
+
+`jslings` can help YOU master Javascript on the terminal
```

Comment:
Quick comment: Maybe you should revert back README for now. 
This file deserves a separate PR to compose, and we are not there yet.